community-plugins.json

```diff
@@ -4558,5 +4558,13 @@
         "description": "Allows you to hide files and folders from the Obsidian file explorer if you don't want to see them anymore.",
         "repo": "Oliver-Akins/file-hider",
         "branch": "main"
+    },
+    {
+        "id": "obsidian-tasks-x-plugin",
+        "name": "Tasks SQL Powered",
+        "author": "Jon Bullen (original by Martin Schenck)",
```

Comment:
I would appreciate that saying `(original by Martin Schenck and Clare Macrae)` please.